server/etc/e-smith/db/configuration/defaults/postfix/IdentityPolicy

```diff
@@ -0,0 +1 @@
+disabled
```

Comment:
I'd change the name of this prop to something which better describe the function.
Could it be something like `SenderRestriction`?

Otherwise if we would like to keep the `IdentityPolicy` name, the values should be more meaningful, something like:
- `permissive`: no check (default, current implementation)
- `strict`: everybody must respect sender policy

Reply:
There's SPF (Sender Policy Framework) already using the word "Sender", whilst "Identity" is used by IMAP clients. I like the suffix Policy though... :thinking: 

Let's go with IdentityPolicy, but think a while about the possible values. If just two are allowed, enabled/disabled is good. I'd opt for other strings if more than two options are planned.

We need to discuss them!

Reply:
> Let's go with IdentityPolicy, but think a while about the possible values. If just two are allowed, enabled/disabled is good. I'd opt for other strings if more than two options are planned.

I agree, if we only have 2 possible values, I'd go with `SenderRestriction` or something similar.

@stephdl what do you think? We will have more than 2 policies in the future?

Reply:
@DavidePrincipi 

for now the name is the same, of course Sender is nice, I modifed and now it is `permissive` and `strict`

Reply:
can we go to SenderPolicy ?

one step to each other :D